libs/storage/src/storage.ts

```diff
@@ -71,6 +71,12 @@ export class Storage<Store> extends EventBus<{
     return this.data;
   }
 
+  async clear() {
+    this.data = {} as Store;
+    await this.commit(this.name, this.data);
+    this.emit('update', this.data);
```

Comment:
You shouldn’t set the default data to `{}` because some storages have a more complex structure than just `{}` (e.g. `NetworkStore`, `BridgeStorage`). You can add a new method to clear the storage in `StorageHandler` and then call that inside this method. Let the client define that function.

Reply:
@AtelyPham @dutterbutter I've updated the `clear` method to `reset(data: Store)`

`
 async reset(data: Store) {
    this.data = data;
    await this.commit(this.name, this.data);
    this.emit('update', this.data);
  }
`

We reset the `Note Storage` like this: `await this.noteStorage.reset({})`

The reason behind not including this function in `StorageHandler` is that we do not have access to `data`(this.data) since it is private not accessible outside the class.